eth/api.go

```diff
@@ -166,8 +166,12 @@ func NewPrivateAdminAPI(eth *Ethereum) *PrivateAdminAPI {
 	return &PrivateAdminAPI{eth: eth}
 }
 
-// ExportChain exports the current blockchain into a local file.
-func (api *PrivateAdminAPI) ExportChain(file string) (bool, error) {
+// ExportChain exports the current blockchain into a local file,
+// or a range of blocks if first and last are non-nil
+func (api *PrivateAdminAPI) ExportChain(file string, first *uint64, last *uint64) (bool, error) {
+	if first != nil && last == nil || first == nil && last != nil {
```

Comment:
Why not let only the `first` be specified and from that point onward export the entire chain till head? Seems more robust to me.

Reply:
You mean when `first` is set and `last` is nil?  Ya, I can see that being useful.  I'll update the PR to support that.

Reply:
Ok, force pushed 947c62b06 which supports this behavior:

```
> admin.exportChain("/tmp/1332997.gz", 1332997, 1333000)
true
> admin.exportChain("/tmp/latest.gz", 1724972)
true
> admin.exportChain("/tmp/bad.gz", null, 1333000)
Error: last cannot be specified without first
    at web3.js:3143:20
    at web3.js:6347:15
    at web3.js:5081:36
    at <anonymous>:1:1
```